nncf/torch/quantization/quantize_model.py

```diff
@@ -169,10 +169,14 @@ def _create_nncf_config(
             compression_config["ignored_scopes"].extend(_ignored_scope)
         else:
             compression_config["ignored_scopes"] = _ignored_scope
+        compression_config["validate_scopes"] = ignored_scope.validate
```

Comment:
It looks like the same changes is needed to the https://github.com/openvinotoolkit/nncf/blob/444ca7da363d8dac82641a0e7851b7b8e7e2e2f4/nncf/tensorflow/quantization/quantize_model.py#L91

Reply:
Yeap, added